src/components/InstancesList/InstancesList.js

```diff
@@ -213,6 +214,12 @@ class InstancesView extends React.Component {
     exportStringToCSV(cqlQuery);
   }
 
+  toggleNewFastAddModal() {
```

Comment:
@K-Felk you may need to change the signature of this function to `toggleNewFastAddModal  = () => {` otherwise the `this` context is I think lost because of the way the function is invoked down below.

Reply:
Yup, good catch-changing the syntax got the modal to open.  I'll re-run tests and linter and then merge if it's clear.  Thanks for your help!